src/components/shared/ActionBanner/index.tsx

```diff
@@ -16,10 +16,12 @@ const ActionBanner = ({
   helpfulText,
   label,
   buttonUnstyled,
-  onClick
+  onClick,
+  ...remainingProps
 }: ActionBannerProps) => {
   return (
-    <div className="action-banner usa-alert">
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <div className="action-banner usa-alert" {...remainingProps}>
```

Comment:
I needed to be able to specify a `data-*` element so that we can select the correct `ActionBanner` to start the business case.